Jenkinsfile

```diff
@@ -322,6 +322,7 @@ boolean skip_scan_rpms_centos7() {
 
 boolean skip_ftest_hw(String size) {
     return env.DAOS_STACK_CI_HARDWARE_SKIP == 'true' ||
+           skip_stage('func-test') ||
```

Comment:
im not sure what this is about?

Reply:
This is a change that just landed in master today. I wonder if somehow git got confused?